sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2040,6 +2040,138 @@ public virtual async Task<AnalyzeHealthcareEntitiesOperation> StartAnalyzeHealth
 
         #endregion
 
+        #region Extract Summary
+
+        /// <summary>
+        /// Performs extractive summarization on the given documents, which consists of extracting
+        /// sentences that collectively represent the most important or relevant information within
+        /// the original content.
+        /// For a list of languages supported by this operation, see
+        /// <see href="https://learn.microsoft.com/en-us/azure/cognitive-services/language-service/summarization/language-support?tabs=document-summarization"/>.
```

Comment:
Never include the locale in links because it won't use the users' browser's preference for language and auto-route:
```suggestion
        /// <see href="https://learn.microsoft.com/azure/cognitive-services/language-service/summarization/language-support?tabs=document-summarization"/>.
```
Please make sure to check for any other instances throughout the PR. I won't repeat for brevity.

Reply:
I'd also elide the query parameters. These aren't required. Just let the default behavior happen.